charts/sn-platform/values.yaml

```diff
@@ -827,6 +834,13 @@ zookeeper:
 ## templates/bookkeeper-statefulset.yaml
 ##
 bookkeeper:
+  # bookkeeperCluster indicates the setup for the CR(CustomResource) of the BookkeeperCluster
+  bookkeeperCluster:
```

Comment:
Shall we use `BookKeeperCluster` to keep consistent with the CR?

Reply:
Fixed.